tests/tests.toml

```diff
@@ -230,8 +230,8 @@ wat = true
 filename = "loop2.py"
 ast = true
 asr = true
-cpp = true
 c = true
```

Comment:
Create `loop3.py` and register it with `c = true`, `asr = true`. Don't de-register a test from `cpp`. 
Also do we really need `ast = true` for `loop2.py` as we are already testing it `asr = true` which will only work if AST is correct, no?

Reply:
Done!